yt/frontends/flash/data_structures.py

```diff
@@ -191,25 +191,31 @@ def __init__(
 
         if self.particle_filename is None:
             # try to guess the particle filename
-            try:
-                self._particle_handle = HDF5FileHandler(
-                    filename.replace("plt_cnt", "part")
-                )
-                self.particle_filename = filename.replace("plt_cnt", "part")
-                mylog.info(
-                    "Particle file found: %s", self.particle_filename.split("/")[-1]
-                )
-            except OSError:
+            if "hdf5_plt_cnt" in filename:
+                # We have a plotfile, look for the particle file
+                try:
+                    pfn = filename.replace("plt_cnt", "part")
+                    self._particle_handle = HDF5FileHandler(pfn)
+                    self.particle_filename = pfn
+                    mylog.info(
+                        "Particle file found: %s", self.particle_filename.split("/")[-1]
```

Comment:
fix Windows portability while we're at it
```suggestion
                        "Particle file found: %s", os.path.basename(self.particle_filename)
```